### PR TITLE
Add template for Issue and Pull Request

### LINKED ISSUE #2632: Support Github Template for Issue and Pull Request
## ⭐️ Brief
It would be nice if we adopt the [Github Template](https://blog.github.com/2016-02-17-issue-and-pull-request-templates/) to make our life easily.

There are couple benefits:
- Our debugging procedure is easier (informative ticket)
- User find the good template to report appropriately

## 👑 Acceptance Criteria
- Adopt Github Template
- Introduce Pull Request Template
- Introduce Issue Template (General report and Bug report)

### PR DESCRIPTION
**Pull request by NghiaTranUIT**
## ❓ What's this?
It's the effort to bring Github Template to our repo and make our life easier.
Inspired and credit for @Toggle's Mobile Team 💯

 ## 🤯 Changelogs 
- Add template for Issue (Bug Report and General Issue) and Pull Request

 ## 👫 Relationships
Closes #2632 

 ## 🔎 Review hints
- Check the grammar or change the word if need

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/b-bug-report.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+name: "🐞 Bug Report"
+about: Bug??? Han, bring me a flamethrower 🚒.
+
+---
+
+## 🐞 Bug Description
+<!-- Short and clear how the bug looks like -->
+<!-- It could be a short converstation with the user -->
+
+## Context
```
Comment (MartinBriza) [inline .github/ISSUE_TEMPLATE/b-bug-report.md]: I think the context should be first so people don't forget it (and it puts Expected Behavior right under Bug Description - these two should be together).

File: .github/ISSUE_TEMPLATE/b-bug-report.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+<!-- Describe the platform and Toggl Version. It's super easy for us to trace back those monter-->
+
+Platform: macOS/Linux/Window
```
Comment (MartinBriza) [inline .github/ISSUE_TEMPLATE/b-bug-report.md]: Windows

File: .github/ISSUE_TEMPLATE/b-bug-report.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+OS Version: macOS 10.14 or Window 10.1
+Toggl Version: 7.4.253
+
+## Expected behavior
+<!-- A clear and concise description of what you expected to happen -->
+
+## To reproduce
+<!-- Clear steps to reproduce the issue -->
```
Comment (MartinBriza) [inline .github/ISSUE_TEMPLATE/b-bug-report.md]: I would make this comment multiline and add a numbered list template so it's clear we want the order of the steps we should take to reproduce this.
Also, "To reproduce" should be something along the lines of "Can this behavior be reproduced every time?" and then let the user describe the steps, if yes.

Comment (NghiaTranUIT) [reply .github/ISSUE_TEMPLATE/b-bug-report.md]: Good point. I will update it 👍

**Comment by MartinBriza** [comment]
Hi! Thanks for the PR, this will be a useful addition to the repository! I see you've already noticed I added a bunch of inline comments to the changes.

In addition to them, I have a few remarks to the change in general - and this is just from my point of view, that doesn't matter everything is necessarily bad:

- I'm missing a feature/enhancement template
- It definitely needs some proofreading (I can do that, I just didn't want to do it in comments), there's a lot of typos or strange formulations. Also, the titles don't have consistent capitalization (e.g. "Bug Description" vs "Expected behavior").
- I think the emojis should be either in all or none of the titles, not just some of them.
- The order of stuff in the PR template is weird - for example, why is Closes # between Relationships and Review Hints? I'd probably go for a different structure altogether in this file.

**Comment by NghiaTranUIT** [comment]
Thank for your feedback @MartinBriza 👍 
I've update the PR in order to fix the typo and add `Improvement/Feature` template.

Regarding "Why Closed # is middle of Relationship and ReviewHint", I leverage the hidden Github's feature to close the Issue automatically after the PR is merged. 

Ref: [Close issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/)

**Pull request merged**